website/docs/api/top-level.md

```diff
@@ -239,7 +239,7 @@ browser. Will run a simple web server.
 | Name      | Description                                                                                                                                                       |
 | --------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `docs`    | Document(s) or span(s) to visualize. ~~Union[Iterable[Union[Doc, Span]], Doc, Span]~~                                                                             |
-| `style`   | Visualization style, `"dep"` or `"ent"`. Defaults to `"dep"`. ~~str~~                                                                                             |
+| `style`   | Visualization style, `"dep"`, `"span"` <Tag variant="new">3.3</Tag> or `"ent"`. Defaults to `"dep"`. ~~str~~                                                                                             |
```

Comment:
Nitpicky, but I think I'd prefer the order `dep`, `ent`, `span` which keeps it alphabetical and makes it easier to see the existing `ent` option, which gets kind of buried behind the new tag in this version.

Reply:
To be honest, I was a little unsure of the order I chose. I have changed it back to alphabetical order